packaging/setup/ovirt_engine_setup/keycloak/constants.py

```diff
@@ -67,7 +67,7 @@ class Const(object):
     OVIRT_ENGINE_KEYCLOAK_SSO_PROFILE = 'internalsso'
     OVIRT_ENGINE_KEYCLOAK_DB_BACKUP_PREFIX = 'keycloak'
     KEYCLOAK_ADD_USER_SCRIPT = 'add-user-keycloak.sh'
-    KEYCLOAK_CLI_ADMIN_SCRIPT = 'kcadm.sh'
+    KEYCLOAK_CLI_ADMIN_SCRIPT = '/usr/bin/timeout -k 60 120 kcadm.sh'
```

Comment:
I don't think this will work - it will probably complain about a `/usr/bin/timeout -k 60 120 kcadm.sh` command missing. You would need to pass this as a list of strings, but current code won't handle this too.

Reply:
According to https://github.com/oVirt/ovirt-engine-keycloak/pull/23 we could pass extraArgs directly to underlying subprocess.popen method which supports timeout https://docs.python.org/3.6/library/subprocess.html

I will try to edit this PR after grooming meeting

Reply:
let's see if it works, always a good test to see if artifacts are picked up correctly...

Reply:
`subprocess.run` does have a `timeout` arg, but `Popen` constructor [doesn't](https://docs.python.org/3.6/library/subprocess.html#popen-constructor). If anything, I would suggest adding that `timeout` wrapping in `kcadm.sh` when running `java` - we already saw `kcadm.sh` failing for verbs other than `config credentials`

Reply:
yay, ost works.
bad news is indeed it needs a list